index.js

```diff
@@ -51,6 +51,12 @@ Plugin.prototype.apply = function(compiler) {
       }
 
       var chunks = {};
+      var outputDir = this.options.path || '.';
+      var outputFilename = path.join(outputDir, this.options.filename || DEFAULT_OUTPUT_FILENAME);
+      var obj = JSON.parse(fs.readFileSync(outputFilename, 'utf8'));
+      if (obj.status == 'done') {
+          chunks = obj.chunks;
+      }
```

Comment:
I tried out your example and verified your fix. Also tried a slightly different implementation. I don't think we can support this.  This issue is that you have multiple webpack configs with multiple instances of BundleTracker. You should be issue each a different name for stats file to each plugin instance and generate 1 stats file per config.

```javascript
        new BundleTracker({path: __dirname, filename: './webpack-stats-' + language + ' .json'}),
```